master/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Master chart that deploys the OS. Targets minikube 
+name: master
```

Comment:
Can we name this something else? I know we don't have an actual project name yet, but `master` seems confusing in multiple ways. How about `platform`?

Reply:
I thought we talked about this before vacation and were going to punt on this until umbrella research had been done. Maybe I remember the inverse.

Reply:
I named it platform for now